src/main/resources/com/mathworks/ci/BuildArtifactAction/summary.jelly

```diff
@@ -7,6 +7,11 @@
   xmlns:i="jelly:fmt">
   <t:summary icon="document.png">
     <p><a href="buildresults">MATLAB Build Results</a></p>
+    <span class="${pst.cssClass}">
+      <j:if test="${it.totalCount == 0}">
+        <font color="#EF2929"><h5> Zero task indicates either none of the task ran from buildfile or MATLAB version does not support the feature</h5></font>
```

Comment:
@mw-hrastega could you please review this comment. This will show on summary only when there are no task run or if task run are 0 due to older MATLAB version

Reply:
Question 1 & 2: 
Answer: Inclusion of plugin explicitly in MATLAB is new feature introduced in MATLAB newer versions and goes back till version that supports buildtool  R2022b

Question 3: 
Answer: No if there was an error in buildfile itself and that did not run any task thats when total task shows 0 task ran . skip tasks will still be counted as skipped task and will be counted under total task. 

![image](https://github.com/mathworks/jenkins-matlab-plugin/assets/47204011/5f6c0937-f10b-409d-b32e-e73146868fd6)

Reply:
The Build result generation is kinda hidden to run build task. User does not have any control on wheather the results should be generated after build run. We generate build results as part build run and this is due to design decision that we took during RF review. Hence showing warnning or error on configuration page may be confusing and missleading wheather he she could run build or not as they still do not know that this will generate build result. Hence we chose to display the error on Summary page which will be shown irrespective of MATLAB version. I like what you suggested i will replace it.

Reply:
Ok, sounds good. Can you also replace the ordered list in the report (1. and 2. ) with an unordered list (bullet points)? We should use an ordered list only if the order matters.